src/redux/categories/Categories.js

```diff
@@ -0,0 +1,18 @@
+const CATEGORY = 'CATEGORY';
```

Comment:
- [ ] (OPTIONAL) Kindly make sure that the values of your action type follow the [ducks pattern](https://github.com/erikras/ducks-modular-redux). Therefore it should be something like this: `const CATEGORY = bookstore/categories/CATEGORY`. Kindly rectify this 💯 
- [ ] (OPTIONAL) Also this should be `CHECK_STATUS` and not `CATEGORY`. It would be great if you could correct that too 👍